File: app/templates/app-java/src/main/java/ui/main/MainPresenter.java
package <%= appPackage %>.ui.main;

import <%= appPackage %>.di.ActivityScope;
import <%= appPackage %>.ui.base.BasePresenter;

import javax.inject.Inject;

@ActivityScope
public class MainPresenter extends BasePresenter<MainView> {

    @Inject
    public MainPresenter() {

    }

}

File: app/templates/app-kotlin/src/main/java/ui/main/MainPresenter.kt
package <%= appPackage %>.ui.main

import <%= appPackage %>.di.ActivityScope
import <%= appPackage %>.ui.base.BasePresenter

import javax.inject.Inject

@Suppress("EmptyClassBlock")
@ActivityScope
class MainPresenter @Inject constructor() : BasePresenter<MainView>() {


}
